### PR TITLE
fix PM not loading on first app launch [SP-3194]

### PR DESCRIPTION
**Pull request by Suryakantj**
Updated javascript injection logic from onPagefinished to onPageStarted

### CODE REVIEW

File: cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/ConsentWebView.java [hunk 1]
```diff
@@ -93,16 +94,21 @@ private void setup() {
...
+            public void onPageStarted(WebView view, String url, Bitmap favicon) {
+                super.onPageStarted(view, url, favicon);
                 try {
                     view.loadUrl("javascript:" + getFileContent(getResources().openRawResource(R.raw.js_receiver)));
                 } catch (IOException e) {
                     ConsentWebView.this.onError(new ConsentLibException(e, "Unable to load jsReceiver into ConasentLibWebview."));
                 }
             }
 
+            @Override
```
Comment (leodoin) [inline cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/ConsentWebView.java:106]: do we have to keep this overriden method here?

Comment (Suryakantj) [reply cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/ConsentWebView.java:106]: Thank you @leodoin

Comment (Suryakantj) [reply cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/ConsentWebView.java:106]: > do we have to keep this overriden method here?

I will test by removing this overridden method , I believe we dont require this method.
I will let you know once testing is done.

Comment (Suryakantj) [reply cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/ConsentWebView.java]: @leodoin Updated javascript injection logic and removed unused

File: cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/ConsentWebView.java [hunk 2]
```diff
@@ -93,13 +94,17 @@ private void setup() {
...
             @Override
-            public void onPageFinished(WebView view, String url) {
-                super.onPageFinished(view, url);
-                //view.loadUrl("javascript:" + "addEventListener('message', SDK.onEvent('oie'))");
+            public void onPageStarted(WebView view, String url, Bitmap favicon) {
+                super.onPageStarted(view, url, favicon);
                 try {
-                    view.loadUrl("javascript:" + getFileContent(getResources().openRawResource(R.raw.js_receiver)));
+                    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
+                        view.evaluateJavascript(getFileContent(getResources().openRawResource(R.raw.js_receiver)),null);
```
Comment (leodoin) [inline cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/ConsentWebView.java:101]: nice addition here :)

**Pull request merged**